src/main/manager/lifecycle-manager.service.ts

```diff
@@ -72,50 +72,49 @@ export class LifecycleManager {
     this.logger.debug(`Enabling mod: ${modId} with ${releaseAssets.length} release assets`)
     for (const releaseAsset of releaseAssets) {
       this.logger.debug(`Enabling release asset: ${releaseAsset.id}`)
-
-      const { baseUrl } = getUrlPartsForDownload(releaseAsset.source)
-
-      let srcPath = posixpath(
-        join(
-          await this.writeDirectoryService.getWriteDirectoryForRelease(subscription, release),
-          releaseAsset.source.replace(baseUrl, '')
+      for (const link of releaseAsset.links) {
+        let srcPath = posixpath(
+          join(
+            await this.writeDirectoryService.getWriteDirectoryForRelease(subscription, release),
+            link.source
+          )
         )
-      )
-
-      // If the source is a hash path, we need to extract the base path and make sure the symlink is for the exploded folder including internal route
-      if (HashPath.isHashPath(srcPath)) {
-        this.logger.debug(`Source is a hash path: ${srcPath}`)
-        const hashPath = new HashPath(srcPath)
-        srcPath = join(hashPath.basePathWithoutExt, hashPath.hashPath)
-      }
 
-      const targetPath = posixpath(
-        await this.variablesService.replaceVariables(releaseAsset.target)
-      )
-      this.logger.log(
-        `Creating Symlink for release asset: ${releaseAsset.id} from ${srcPath} to ${targetPath}`
-      )
-      ensureDirSync(dirname(targetPath))
-
-      if (await pathExists(targetPath)) {
-        this.logger.error(
-          `Target path already exists: ${targetPath}, please remove it and try again`
-        )
-        throw new Error(`Target path already exists: ${targetPath}, please remove it and try again`)
-      }
+        // If the source is a hash path, we need to extract the base path and make sure the symlink is for the exploded folder including internal route
```

Comment:
If we have removed hash paths what does this refer to?

Reply:
Nothing removed